### DISCUSSION TITLE
part-ii/2021-137/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#363] [Question comments]
Part II 2021 Riemann Surfaces: Paper 3, Section II, F 

https://questions.tripos.org/part-ii/2021-137/

**Comment by artie2000** [comment]
<details>
<summary>(a)</summary>

Apply Riemann-Hurwitz to $\bar{f}:\mathbb{C}_\infty\to\mathbb{C}_\infty$ (ramification at infinity and degree are bookwork).

For the example, set $f'(z)=\prod_{j=1}^k(z-j)^{m_j-1}$, and take the polynomial antiderivative.

</details>

(b) is bookwork, or on 2021 example sheets.

<details>
<summary>(c)</summary>

Important facts: composition of paths induces composition of permutations, and change of basepoint induces conjugation of permutations.

Let $z_0$ be a branch point with preimages $\{w_i\}$. Lifting a loop $t\to e^{2\pi i t}$ in regular $\{z\to z^{m_f(w_i)}\}$ charts around $z_0$ induces a permutation of cycle type $(m_f(w_1))\dots(m_f(w_k))$.

Suppose such an $f$ exists. Then small loops around 0 and 1 induce double transpositions, while a small loop around $\infty$ induces a 3-cycle. Changing to a common basepoint, the product of the loops around 0 and 1 is homotopic to the loop around infinity, contradicting the hint.

</details>
